LICENSE.md

```diff
@@ -1,5 +1,5 @@
-Copyright (c) 2014-2020, Sideway Inc, and project contributors  
-Copyright (c) 2014, Walmart.  
+Copyright (c) 2014-2021, Sideway Inc, and project contributors
```

Comment:
This is outside the scope of this patch. And should it really extend "Sideway Inc" copyright like that? Hasn't it been transferred away?

Reply:
I wanted to extend the copyright to project contributors (i.e. myself with this patch) into 2021.  I will add it on a new line to represent the fact that Sideway hasn't contributed in 2021.